@xen-orchestra/web/src/components/pool/PoolNetworkSidePanel.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <UiPanel>
+    <template #header>
+      <UiButton
+        v-tooltip="$t('coming-soon')"
+        disabled
+        variant="tertiary"
+        size="medium"
+        accent="brand"
+        :left-icon="faEdit"
+      >
+        {{ $t('edit') }}
+      </UiButton>
+      <UiButton
+        v-tooltip="$t('coming-soon')"
+        disabled
+        variant="tertiary"
+        size="medium"
+        accent="danger"
+        :left-icon="faTrash"
+      >
+        {{ $t('delete') }}
+      </UiButton>
+      <UiButtonIcon v-tooltip="$t('coming-soon')" disabled accent="brand" size="medium" :icon="faEllipsis" />
+    </template>
+    <template #default>
+      <UiCard class="card-container">
+        <UiCardTitle v-tooltip="{ placement: 'bottom-end' }" class="typo-body-bold text-ellipsis">
+          {{ network.name_label }}
+        </UiCardTitle>
+        <div class="content">
+          <!-- ID -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('id') }}
+            </template>
+            <template #value>
+              <div class="value">
+                {{ network.id }}
+              </div>
+            </template>
+            <template v-if="network.id" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                accent="brand"
+                size="medium"
+                :icon="faCopy"
+                @click="copy(network.id)"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- DESCRIPTION -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ $t('description') }}</template>
+            <template #value>
+              <div class="value">
+                {{ network.name_description }}
+              </div>
```

Comment:
Let's prefer `span` over `div` in these `#value` templates

```suggestion
              <span class="value">{{ network.name_description }}</div>
```